src/api.rs

```diff
@@ -181,7 +181,7 @@ impl Context {
   pub fn frame_properties(&mut self, idx: u64) -> bool {
     let key_frame_interval: u64 = self.fi.config.key_frame_interval;
 
-    let reorder = self.fi.config.low_latency;
+    let reorder = !self.fi.config.low_latency;
```

Comment:
So, now low_latency == true means, encode in low latency and don't use frame reordering, correct?

Reply:
yes